src/pal/prebuilt/inc/corprof.h

```diff
@@ -438,50 +438,50 @@ typedef struct _COR_PRF_METHOD
     mdMethodDef methodId;
     }   COR_PRF_METHOD;
 
-typedef void __stdcall __stdcall FunctionEnter( 
+typedef void STDMETHODCALLTYPE STDMETHODCALLTYPE FunctionEnter(
```

Comment:
`STDMETHODCALLTYPE` should not be duplicated here (I know it was duplicated before...). Could you please fix it?